--- lked.orig	2022-02-06 13:11:30.000000000 +0900
+++ lked	2022-02-06 13:08:29.000000000 +0900
@@ -443,6 +443,12 @@
          set BLAS=' '
          breaksw
 
+      case blas:
+         set MATHLIBS="-llapack -lblas"
+         set BLAS=' '
+         set LAPACK=' '
+         breaksw
+
       case none:
       default:
          echo "Warning.  No math library was found, you should install one."
@@ -864,6 +870,12 @@
          set BLAS=' '
          breaksw
 
+      case blas:
+         set MATHLIBS="-llapack -lblas"
+         set BLAS=' '
+         set LAPACK=' '
+         breaksw
+
       case none:
       default:
          echo "Warning.  No math library was found, you should install one."
