--- rungms.orig	2022-02-06 13:11:14.000000000 +0900
+++ rungms	2022-02-06 13:10:46.000000000 +0900
@@ -81,9 +81,9 @@
 #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
 #
 set TARGET=sockets
-set SCR=~/gamess/restart
-set USERSCR=~/gamess/restart
-set GMSPATH=~/gamess
+set SCR=$HOME/scr
+set USERSCR=$HOME/scr
+set GMSPATH=$HOME/share/gamess
 #
 # Get any MDI-related options and remove them from the argument list
 #
