*** rungms.orig	2019-12-24 19:49:04.155134287 +0900
--- rungms	2019-12-24 19:49:54.903118061 +0900
***************
*** 62,70 ****
  #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
  #
  set TARGET=sockets
! set SCR=/scr1/$USER
! set USERSCR=~/gamess-devv
! set GMSPATH=~/gamess-devv
  #
  set JOB=$1      # name of the input file xxx.inp, give only the xxx part
  set VERNO=$2    # revision number of the executable created by 'lked' step
--- 62,70 ----
  #       See also a very old LoadLeveler "ll-gms" for some IBM systems.
  #
  set TARGET=sockets
! set SCR=$HOME/scr
! set USERSCR=$HOME/scr
! set GMSPATH=$HOME/share/gamess
  #
  set JOB=$1      # name of the input file xxx.inp, give only the xxx part
  set VERNO=$2    # revision number of the executable created by 'lked' step
